app/src/main/java/com/ably/tracking/demo/publisher/PublisherService.kt

```diff
@@ -15,17 +16,11 @@ class PublisherService : Service() {
     // SupervisorJob() is used to keep the scope working after any of its children fail
     private val scope = CoroutineScope(Dispatchers.Main + SupervisorJob())
 
-    private val binder = Binder()
-
     private val notificationProvider: NotificationProvider by inject()
 
     private val assetTracker: AssetTracker by inject()
 
-    inner class Binder : android.os.Binder() {
-        fun getService(): PublisherService = this@PublisherService
-    }
-
-    override fun onBind(intent: Intent?): IBinder = binder
+    override fun onBind(intent: Intent?): IBinder = Binder()
```

Comment:
Won't it fail your builds? :thinking: The custom `Binder` has the `getService()` method and the Android one does not :thinking: